src/cross.ts

```diff
@@ -307,14 +307,15 @@ export const accountDataToAccount = ({
     type === "libcore" ||
     // FIXME Dirty hack, since we have no way here to know if "xpubOrAddress" is one or the other.
     // Proposed fix: https://ledgerhq.atlassian.net/browse/LL-7437
-    currency.family === "bitcoin" ||
-    currency.family === "tezos"
+    currency.family === "bitcoin"
   ) {
     // In libcore implementation, xpubOrAddress field always go in the xpub
     // In JS implementation, only Bitcoin-like currencies store the xpub
     xpub = xpubOrAddress;
   } else {
-    if (!freshAddress) {
+    if (currency.family === "tezos") {
```

Comment:
:see_no_evil: we need to improve that `xpubOrAddress` mechanism :thinking:

Reply:
yeah, we need that `importAccount(id)` bridge hook :)